src/frontend/src/components/Map.js

```diff
@@ -146,6 +146,7 @@ const Map = ({
   onAddAnnotationHexagonIDs,
   hexResolution,
 }) => {
+  console.log('Render map');
```

Comment:
were these debug logs or are they needed?

Reply:
I wanted to see what we were loading to the screen, these are render logs I think it's good to hold because I see a lot of renders and potential improvements